docsSource/sections/01-usage.md

```diff
@@ -8,13 +8,18 @@ Install with npm:
 $ npm install animate.css --save
 ```
 
-with yarn:
+Or install with Yarn:
 
 ```shell
 $ yarn add animate.css
 ```
 
-or add it directly to your webpage using a CDN:
+Import it into your file:
+```js
+import "animate.css"
+```
+
```

Comment:
We would also need a note about having to use a bundler or other tools to make this work as it might cause some confusion and lots of new issues in our tracker.

Reply:
Hey @eltonmesquita sorry I don't understand what you mean re: bundler or other tools.

I just re-tested these steps for `create react-app` and they all work out of the box:
```
yarn create react-app my-app
cd my-app/
yarn add animate.css
yarn start
...
Add import "animate.css" (e.g. in App.js)
```

Keeping the steps simple (and agnostic) should cause the least amount of confusion?

Reply:
As this library is used by all levels of devs, there'll probably be lots of people that have no idea that you need tooling for importing CSS in a js environment. Something like "_*This will only work with appropriate tooling like webpack, parceljs and others. Check the CDN usage if you are not using any tool for packing or bundling your code._" should do.

Reply:
@eltonmesquita Updated per your comments. Re-ran docs. Thanks!